src/xdocs/config_system_properties.xml

```diff
@@ -33,6 +33,76 @@
         is <a href="property_types.html#boolean">boolean</a> and defaults
         to <code>false</code>.
       </p>
+      <p>
+        Example of usage by xml include to let keep common part of configs in single file:
```

Comment:
Similar to my last comment, I think this needs to be explained a bit more about how each piece of the config is different and why someone would use this.

Reply:
did some changes